### ISSUE TITLE
CBF CIFTI project failing

### ISSUE THREAD

**Issue by tsalo** [PennLINC/aslprep#360]
## Summary

CIFTI outputs from #338 aren't being generated correctly. Unfortunately, the easiest way for me to debug this is to merge the PR, build the Singularity image on CUBIC, and test there.

## Additional details

- ASLPrep version: 0.6.0rc3 (post #338)
- Docker version:
- Singularity version:

### What were you trying to do?

Include `--cifti-output` in the ASLPrep call.

### What did you expect to happen?

Get CIFTI outputs.

### What actually happened?

```
Stderr:

	While running:
	wb_command -volume-to-surface-mapping /src/aslprep/.circleci/work/test_003_full/aslprep_0_6_wf/sub_01_wf/asl_preproc_ses_1_wf/ds_asl_cifti_wf/warp_mean_cbf_to_anat/sub-01_ses-1_asl_DeltaMOrCBF_meancbf_trans.nii.gz /src/aslprep/.circleci/data/anatomical/smriprep/sub-01/anat/sub-01_hemi-L_midthickness.surf.gii sub-01_hemi-L_midthickness.surf_mapped.func.gii -ribbon-constrained /src/aslprep/.circleci/data/anatomical/smriprep/sub-01/anat/sub-01_hemi-L_white.surf.gii /src/aslprep/.circleci/data/anatomical/smriprep/sub-01/anat/sub-01_hemi-L_pial.surf.gii -volume-roi /src/aslprep/.circleci/work/test_003_full/aslprep_0_6_wf/sub_01_wf/asl_preproc_ses_1_wf/asl_cifti_resample_wf/goodvoxels_bold_mask_wf/goodvoxels_mask/sub-01_ses-1_asl_reduced_std_maths_maths_maths_thresh_maths.nii.gz

	ERROR: roi volume is not in the same volume space as input volume
```

**Comment by tsalo** [comment]
Should have been closed by #361.

**Comment by tsalo** [comment]
The problem in this case was that I was passing an aslref-space ASL image to the projection step instead of an anatomical-space ASL image.
